### ISSUE TITLE
Cannot built by vite

### ISSUE THREAD

**Issue by ALiangLiang** [henk-badenhorst/vue-hotjar-next#4]
I am run normally in vite development mode successful. But can't built.

```
vite v2.3.8 building for production...
✓ 645 modules transformed.
'default' is not exported by node_modules/vue-hotjar-next/index.js, imported by src/main.js
file: <PROJECT_ROOT>/src/main.js:3:7
1: import { createApp } from 'vue'
2: import { createGtm } from '@gtm-support/vue-gtm'
3: import Hotjar from 'vue-hotjar-next'
          ^
4: import firebase from 'firebase/app'
5: import 'firebase/firestore'
error during build:
Error: 'default' is not exported by node_modules/vue-hotjar-next/index.js, imported by src/main.js
    at error (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:164:30)
    at Module.error (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:9883:16)
    at Module.traceVariable (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:10264:29)
    at ModuleScope.findVariable (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:9041:39)
    at Identifier.bind (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:4572:40)
    at CallExpression.bind (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:2891:31)
    at CallExpression.bind (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:6989:15)
    at ExpressionStatement.bind (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:2895:23)
    at Program.bind (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:2891:31)
    at Module.bindReferences (<PROJECT_ROOT>/node_modules/rollup/dist/shared/rollup.js:9879:18)
```

**Comment by henk-badenhorst** [comment]
@ALiangLiang Thanks for reporting. This plugin is still a work in progress. I will publish a mature and document version within the next two weeks. I will be sure to test it with Vite.

**Comment by paulschraven** [comment]
Any updates on when you think you will release the mature version @henk-badenhorst? We're waiting for it before implementing :)

**Comment by henk-badenhorst** [comment]
Hi @paulschraven and @ALiangLiang sorry for only getting around to fix this now. I have published a new [version (1.2.0)](https://github.com/henk-badenhorst/vue-hotjar-next/pull/5#issuecomment-894730619). I did test this with Vite and works as expected. I will be making some minor updates in the coming week but this version is ready for production. I have not yet added the Hotjar API Interface in this release. I need some time to figure out what is the best way to include this. If you have any issues please let me know.

**Comment by henk-badenhorst** [comment]
I am closing this issue due to inactivity. Please create a new issue if you have any problems with version 1.2.0
